cogs/verification.py

```diff
@@ -0,0 +1,126 @@
+# Std Lib Imports
```

Comment:
similarly; what is this file for?

Reply:
The idea with this verification system was we would start tracking discord user to steam account, I did not think it was in this PR.